src/ingest/__init__.py

```diff
@@ -21,6 +21,7 @@
 API_URL = os.getenv('API_URL', 'http://localhost:8080/')
 DATASET_HARVESTER_BASE_URI = os.getenv('DATASET_HARVESTER_BASE_URI', 'http://localhost:8080/dataset')
 FDK_DATASERVICE_HARVESTER_URI = os.getenv('FDK_DATASERVICE_HARVESTER_URI', 'http://localhost:8080/dataservice')
+FDK_MODEL_HARVESTER_URI = os.getenv('FDK_MODEL_HARVESTER_URI', 'http://localhost:8080/infomodel')
```

Comment:
Bør FDK_MODEL_HARVESTER_URI være uten path? Da kan du legge til '/infomodel/catalogs/' der den brukes.

Reply:
Den er egentlig uten path, /infomodel finnes ikke i staging/demo/prod. Bruker infomodel for å skille den fra de to andre høsterene i wiremock, ellers har vi 3 mocker som krangler om /catalogs.